src/pymor/vectorarrays/mpi.py

```diff
@@ -102,6 +102,20 @@ def amax(self):
     def __del__(self):
         mpi.call(mpi.remove_object, self.obj_id)
 
+    @property
+    def real(self):
+        real_id = mpi.call(_MPIVectorArray_real, self.obj_id)
+        return MPIVectorArray(real_id, self.space)
+
+    @property
+    def imag(self):
+        imag_id = mpi.call(_MPIVectorArray_imag, self.obj_id)
+        return MPIVectorArray(imag_id, self.space)
+
+    def conj(self):
+        conj_id = mpi.call(_MPIVectorArray_conj, self.obj_id)
```

Comment:
Since `conj` is not a property, you can also use `mpi.method_call_manage`  here.